gateway/conf.d/apicast.conf

```diff
@@ -9,6 +11,12 @@ set_by_lua_block $deployment {
 # ssl_session_fetch_by_lua_block { require('apicast.executor').call() }
 # ssl_session_store_by_lua_block { require('apicast.executor').call() }
 
+{% if jaeger_enabled %}
```

Comment:
This is trouble. There is no liquid templating in this file. This global definition can be in the nginx.conf.liquid.